### ISSUE TITLE
Accessing .components of an object without any causes segfault

### ISSUE THREAD

**Issue by sdfgeoff** [UPBGE/blender#338]
Blend:
http://pasteall.org/blend/index.php?id=44650

It may also be something to do with accessing it inside a collision callback. Not sure which

**Comment by youle31** [comment]
Hi, I can't reproduce the crash with last master with projectile.py neither with damage.py

**Comment by sdfgeoff** [comment]
For me in that blend when the two objects interact (and it tries to print hit_object.components, blender crashes.

crash.txt:

```
# Blender 2.78 (sub 4), Commit date: 2016-12-09 18:10, Hash 5ae8cbd

# backtrace
bin/upbge(BLI_system_backtrace+0x1d) [0x1327fdd]
bin/upbge() [0x9cfcc9]
/usr/lib/libc.so.6(+0x330b0) [0x7f99f443d0b0]
bin/upbge(_ZN13KX_GameObject21pyattr_get_componentsEPvPK18KX_PYATTRIBUTE_DEF+0x9) [0x14e5b99]
bin/upbge(_ZN12PyObjectPlus14py_get_attrdefEP7_objectPK18KX_PYATTRIBUTE_DEF+0x1f5) [0x15af675]
/usr/lib/libpython3.5m.so.1.0(+0x8406e) [0x7f99f935506e]
/usr/lib/libpython3.5m.so.1.0(_PyObject_GenericGetAttrWithDict+0x7e) [0x7f99f9383efe]
/usr/lib/libpython3.5m.so.1.0(PyEval_EvalFrameEx+0x295e) [0x7f99f93f29fe]
/usr/lib/libpython3.5m.so.1.0(+0x1285e3) [0x7f99f93f95e3]
/usr/lib/libpython3.5m.so.1.0(PyEval_EvalCodeEx+0x23) [0x7f99f93f96c3]
/usr/lib/libpython3.5m.so.1.0(+0x92fd8) [0x7f99f9363fd8]
/usr/lib/libpython3.5m.so.1.0(PyObject_Call+0x47) [0x7f99f93392b7]
/usr/lib/libpython3.5m.so.1.0(+0x7ed14) [0x7f99f934fd14]
/usr/lib/libpython3.5m.so.1.0(PyObject_Call+0x47) [0x7f99f93392b7]
bin/upbge(_Z21RunPythonCallBackListP7_objectPS0_jj+0x18a) [0x15b28aa]
bin/upbge(_ZN13KX_GameObject21RunCollisionCallbacksEPS_R28KX_CollisionContactPointList+0x10a) [0x14e591a]
bin/upbge(_ZN24KX_CollisionEventManager9NextFrameEv+0x171) [0x151b591]
bin/upbge(_ZN16SCA_LogicManager10BeginFrameEdd+0x40) [0x1555ea0]
bin/upbge(_ZN15KX_KetsjiEngine9NextFrameEv+0x2a9) [0x14eff99]
bin/upbge(_ZN11LA_Launcher15EngineNextFrameEv+0x5d) [0x14712cd]
bin/upbge(_ZN11LA_Launcher14EngineMainLoopEv+0x41) [0x1471421]
bin/upbge(StartKetsjiShell+0x192) [0x14701e2]
bin/upbge() [0xac0efb]
bin/upbge() [0x9d7452]
bin/upbge() [0x9d875d]
bin/upbge() [0x9d8be8]
bin/upbge() [0x9d9078]
bin/upbge(wm_event_do_handlers+0x3b5) [0x9d9565]
bin/upbge(WM_main+0x18) [0x9d0848]
bin/upbge(main+0x3e6) [0x9b2a16]
/usr/lib/libc.so.6(__libc_start_main+0xf1) [0x7f99f442a291]
bin/upbge(_start+0x2a) [0x9cccaa]
```

**Comment by panzergame** [comment]
fixed in ee07ebaa53e1fc0709ac356995bdd654de023472.
